reference/uitpas.json

```diff
@@ -355,12 +383,59 @@
                       },
                       "eventId": "31e926e2-a35f-11eb-bcbc-0242ac130002",
                       "regularPrice": 10,
-                      "regularPriceLabel": "Optional descriptive label of the regular price. Used in financial reporting.",
-                      "numberOfTickets": 5
+                      "regularPriceLabel": "Optional descriptive label of the regular price. Used in financial reporting."
+                    },
+                    {
+                      "uitpasNumber": "0560002524314",
+                      "tariff": {
+                        "id": "SOCIALTARIFF"
+                      },
+                      "eventId": "31e926e2-a35f-11eb-bcbc-0242ac130002",
+                      "regularPrice": 10,
+                      "regularPriceLabel": "Optional descriptive label of the regular price. Used in financial reporting."
+                    },
+                    {
+                      "uitpasNumber": "0560002524314",
+                      "tariff": {
+                        "id": "SOCIALTARIFF"
+                      },
+                      "eventId": "31e926e2-a35f-11eb-bcbc-0242ac130002",
+                      "regularPrice": 10,
+                      "regularPriceLabel": "Optional descriptive label of the regular price. Used in financial reporting."
+                    },
+                    {
+                      "uitpasNumber": "0560002524314",
+                      "tariff": {
+                        "id": "SOCIALTARIFF"
+                      },
+                      "eventId": "31e926e2-a35f-11eb-bcbc-0242ac130002",
+                      "regularPrice": 10,
+                      "regularPriceLabel": "Optional descriptive label of the regular price. Used in financial reporting."
+                    },
+                    {
+                      "uitpasNumber": "0560002524314",
+                      "tariff": {
+                        "id": "SOCIALTARIFF"
+                      },
+                      "eventId": "31e926e2-a35f-11eb-bcbc-0242ac130002",
+                      "regularPrice": 10,
+                      "regularPriceLabel": "Optional descriptive label of the regular price. Used in financial reporting."
+                    }
+                  ]
+                },
+                "Example using a reward tariff": {
+                  "value": [
+                    {
+                      "tariff": {
+                        "id": "REWARD1234"
+                      },
+                      "uitpasNumber": "0560002524314",
+                      "eventId": "31e926e2-a35f-11eb-bcbc-0242ac130002",
+                      "regularPrice": 10
                     }
                   ]
                 }
-              }
+              }              
```

Comment:
There is some unnecessary trailing whitespace here

Reply:
```suggestion
              }
```